### DISCUSSION TITLE
User Abuse

### DISCUSSION THREAD

**Discussion by VillainsRule** [npm/feedback#856] [General]
[This](https://www.npmjs.com/~79w) user seems to have spam created packages.
One of their spammed packages has a name I'd like.
Could I get the account termed to get the package name, or is there no regulation on package spam?

**Comment by MylesBorins** [comment]
In general you can report spam accounts via our support form

https://www.npmjs.com/support

Looking at the account some of the packages appear to be legitimate and not spam, we'd need a member of our trust + safety team to review the request.

For package naming disputes please follow the process as outlined [here](https://docs.npmjs.com/policies/disputes)

**Comment by Thanga-Ganapathy** [comment]
@MylesBorins How come this user is not a spam, if they had few legitimate pkgs doesn't mean they are not spamming, 

Please have a look at this user published pkgs, it is more than 500.

Looking at the pkgs source from code tab, the index.js file is `0` bytes.

Please look at the published pkg names, mostly 3 chars, which means catchy names were taken. as the OP like users wants that.

Finaly looking at the user published pkg names pattern with no code in it, then action must be taken immediately.

I hope you do ASAP.
